doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -205,6 +205,7 @@ Libraries for networking
 ------------------------
 * :ref:`lib_fota_download` library:
   * Skipping host name check when connecting to TLS service using just IP address.
+  * Bootloader FOTA download standardized to accept only full dual pathnames (e.g. "path/to/s0.bin path/to/s1.bin"). Truncated dual pathnames ("path/to/s0.bin s1.bin") no longer supported.
```

Comment:
```suggestion
  * Bootloader FOTA download standardized to accept only full dual path names (for example "path/to/s0.bin path/to/s1.bin").
  Truncated dual path names ("path/to/s0.bin s1.bin") no longer supported.
```
Do not use latin abbreviations.
Every sentence on its own line.

Reply:
Done and done!